include/swift/Runtime/Concurrency.h

```diff
@@ -41,6 +41,21 @@
 #endif
 #endif
 
+// Does the runtime provide priority escalation support?
+//
+// TODO (rokhinip): Is there a more robust/explicit way to determine platforms
+// which support 128 bit DCAS?
+#ifndef SWIFT_CONCURRENCY_ENABLE_PRIORITY_ESCALATION
```

Comment:
Is there a more robust/explicit way to determine platforms which support 128 bit DCAS?

Reply:
Once we move to C++17, we could probably figure it out with `std::atomic<T>::is_always_lock_free`, but we can't use that yet.